internal/etag/etag.go

```diff
@@ -263,6 +263,12 @@ func FromContentMD5(h http.Header) (ETag, error) {
 	return ETag(b), nil
 }
 
+// IsNotContentMD5Requested - for http.request.header is not request Content-Md5
+func IsNotContentMD5Requested(h http.Header) bool {
```

Comment:
We can keep the function name a bit simpler

```
func ContentMD5Requested(h http.Header) bool {
   return ok
```

Now use this as inverse in the check

```
!etag.ContentMD5Requested(r.Header) && (crypto.S3KMS.IsRequested(r.Header) || crypto.SSEC.IsRequested(r.Header)) {
```